src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* **Breaking change** `http.target` replaced with `http.route`
```

Comment:
Would you describe this in detail here itself? 
For example, this is only applicable to metrics. And that this changes the metric named "http.duration", by renaming a dimension from x to y.